pwnlib/crypto/__init__.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+
+# load cipher suites
+from pwnlib.crypto.ciphers.atbash import cipher_atbash
+from pwnlib.crypto.ciphers.bacon import cipher_bacon
+from pwnlib.crypto.ciphers.base64 import cipher_base64
+from pwnlib.crypto.ciphers.binary import cipher_binary
+from pwnlib.crypto.ciphers.caesar import cipher_caesar
+from pwnlib.crypto.ciphers.decimal import cipher_decimal
+from pwnlib.crypto.ciphers.hex import cipher_hex
+from pwnlib.crypto.ciphers.morse import cipher_morse
+from pwnlib.crypto.ciphers.reverse import cipher_reverse
+from pwnlib.crypto.ciphers.rot13 import cipher_rot13
+from pwnlib.crypto.ciphers.transposition import cipher_transposition
+from pwnlib.crypto.ciphers.vigenere import cipher_vigenere
+from pwnlib.crypto.ciphers.xor import cipher_xor
+
+class crypto():
+    def __init__(self, algo=None, key=None):
+        cipher = None
+        algo = algo
+        key = key
+
+        # select cipher suite
+        if(algo == 'atbash'):
```

Comment:
This is not C nor Java :)
We try to keep code format in line with [PEP8](https://www.python.org/dev/peps/pep-0008/)
```suggestion
        if algo == 'atbash':
```

Reply:
Thanks for your suggestions.

I will try to consider them the next days.